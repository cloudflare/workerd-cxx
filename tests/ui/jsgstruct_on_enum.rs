#[cxx::bridge]
mod ffi {
    #[derive(JsgStruct)]
    enum MyEnum {
        Variant1,
        Variant2,
    }
}

fn main() {}