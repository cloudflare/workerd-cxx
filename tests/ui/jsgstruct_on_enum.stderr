error: JsgStruct only applies to structs, not enums
 --> tests/ui/jsgstruct_on_enum.rs:3:14
  |
3 |     #[derive(JsgStruct)]
  |              ^^^^^^^^^